.buildkite/pipeline.yml

```
@@ -0,0 +1,16 @@
+steps:
+  - label: ":shell: Tests"
+    plugins:
+      docker-compose#v4.8.0:
```

latest version is 4.9.0 now:
```suggestion
      docker-compose#v4.9.0:
```